shared/src/main/scala/com/nitro/nmesos/config/Validations.scala

```diff
@@ -26,9 +26,19 @@ object Validations extends ValidationHelper {
     check("Resources - Num Instances", "Must be > 0") {
       env.singularity.slavePlacement.exists(_ == "SPREAD_ALL_SLAVES") || env.resources.instances.exists(_ > 0)
     },
-    checkWarning("Container - Ports", "No ports defined") {
+    checkWarning("Container - Ports", "Should be defined in HOST mode") {
       env.container.network.exists(_ == "HOST") || !env.container.ports.toSet.flatten.isEmpty
     },
+    check("Container - Port values", "Must be >= 0 and <= 65535") {
+      env.container.ports match {
+        case Some(ports) => ports.map(portMap => portMap.hostPort match {
```

Comment:
Works, but too ugly.
you can replace it by something like:
```
env.container.ports.toSeq.flatten.forall(isValidPort)
```

with the helper in `ValidationHelper`:
```
  def isValidPort(port:PortMap) =  isValidPort(port.containerPort) && port.hostPort.map(isValidPort).getOrElse(true)
  def isValidPort(port:Int) =  0 <= port && port <= 65535
```

Reply:
Ah, clever! :) Fixing now.